src/main/java/nextstep/courses/domain/SessionUserMapping.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.courses.domain;
+
+import java.time.LocalDateTime;
+
+public class SessionUserMapping {
+    private Long id;
```

Comment:
```suggestion
    private final Long id;
```
가능하면 인스턴스 변수의 경우 final로 구현하고, 값을 변경해야 하는 경우에만 final을 제거하면 어떨까?